### DISCUSSION TITLE
Profile doesn't exist..

### DISCUSSION THREAD

**Discussion by steenblikrs** [beakerbrowser/beaker#1732] [General]
I started the drive when I updated my browser, but two things interupted the setup process, it gave me a notification that I had old drives(asked me to convert them), and I saw the template page, and I clicked on one of the templates to create my new hyperdirve listed below. The problem is that I now don't have a beaker profile. How do I create one after the initial setup has been completed?

hyper://85db788f9a281b94a2c04422cc8ecb1bd2375bfa623431a70b867b8a9222c3d6/

![image](https://user-images.githubusercontent.com/19368079/84196411-61efb000-aa5d-11ea-87de-54739fe2e2bf.png)

**Comment by steenblikrs** [comment]
Later I went into the address book and tried to create a profile that way, but when I put my hyperdrive address it never creates the contact let alone a profile.

I also don't have access to the Blahbity-blog because I don't have a profile..

Should I try reinstalling Beaker? If I do that will I still have my existing hyperdrives?

**Reply by pfrazee** [reply]
Yeah this is a weird one. Clearly your setup got hosed -- have you restarted beaker since the first run? (As in you're sure it's not running in the tray.) If not, try that and see if it tries to create your profile again during startup.

If not, we can try to manually fix this. Create a drive that will be your profile, then go to `beaker://system/` (EDIT: no sorry, `hyper://system/`) and find the `address-book.json` file. If that fille doesn't exist, create it. Mine looks like of like this:

```
{
  "profiles": [
    {
      "key": "6900790c2dba488ca132a0ca6d7259180e993b285ede6b29b464b62453cd5c39"
    }
  ],
  "contacts": []
}
```

Change the "key" value to be the key of the drive you created to be your profile. You can find the key in the URL:

```
hyper://6900790c2dba488ca132a0ca6d7259180e993b285ede6b29b464b62453cd5c39/
        ^---------------------the key----------------------------------^
```

Then restart beaker and you should be set.

**Reply by todrobbins** [reply]
Apologies for just popping in, but I don't see anything myself when I navigate to `beaker://system/`:

<img width="1008" alt="Screen Shot 2020-06-10 at 11 49 06 AM" src="https://user-images.githubusercontent.com/158590/84301045-6a54f300-ab10-11ea-9320-f2fda0383b36.png">

<img width="1008" alt="Screen Shot 2020-06-10 at 11 49 12 AM" src="https://user-images.githubusercontent.com/158590/84301050-6d4fe380-ab10-11ea-9eb4-92513732d061.png">

**Reply by pfrazee** [reply]
Oh shoot I mis-spoke, it's `hyper://system/`

**Comment by steenblikrs** [comment]
Thanks guys for your help!!

Ok, so I had thought I had restarted the browser, but after reading your responses, realized it's always running in the background and thus simply closing the browser doesn't quite work. After going to the system tray and restarting, it did ask me to make a profile. Now that I have it, I'd really like my Photo Journal my default profile address. Is rthere a place I can go to change the address associated with my profile? It doesn't give you the option during setup?

**Reply by pfrazee** [reply]
You _should_ be able to follow the process I described where you manually modify `address-book.json` and then restart the browser. Be sure to replace the current one, not just add your new drive to profiles.

**Reply by steenblikrs** [reply]
Thank you very much! That worked like a charm! I am now in the User Directory under steenblikrs! Thanks guys!

**Comment by pfrazee** [comment]
Good deal! Sorry for the trouble
